docs/source/monitoring.rst

```diff
@@ -40,9 +40,9 @@ Nagios' website_.
 
 After installing, simply replace the configuration files with those found in this repository.
 
```

Comment:
```suggestion
```
I think this is referencing the now private repo

Reply:
Good catch, thanks